aim/cli/up/utils.py

```diff
@@ -11,11 +12,13 @@ def build_db_upgrade_command():
         ini_file = os.path.join(migrations_dir, 'alembic.ini')
     else:
         ini_file = os.path.join(migrations_dir, 'alembic_dev.ini')
-    return ['alembic', '-c', ini_file, 'upgrade', 'head']
+    return [sys.executable, '-m', 'alembic', '-c', ini_file, 'upgrade', 'head']
 
 
 def build_uvicorn_command(host, port, num_workers, ssl_keyfile, ssl_certfile, log_level):
-    cmd = ['uvicorn', '--host', host, '--port', '%s' % port, '--workers', '%s' % num_workers]
+    cmd = [sys.executable, '-m', 'uvicorn',
+           '--host', host, '--port', '%s' % port,
+           '--workers', '%s' % num_workers]
```

Comment:
`'%s'`  - this is just completely obsolete

Reply:
it actually is not, as `subprocess` requires all the list members to be `str` values

Reply:
replace with f-string?
`f'num_workers'`